src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -103,6 +100,10 @@ export const useInfiniteTransactionList = ({
     estimateSize: (i) =>
       typeof formattedTransactions[i] === 'string' ? 34 : 52,
     overscan: 20,
+    getItemKey: (i) => {
+      const txOrLabel = formattedTransactions[i];
+      return typeof txOrLabel === 'string' ? txOrLabel : txOrLabel.hash;
+    },
```

Comment:
this should fix some UI shifts in the list that sometimes it bugs out when a tx changes state